src/redux/reducers/genes/index.js

```diff
@@ -3,7 +3,7 @@ import {
   GENES_PROPERTIES_LOADING, GENES_PROPERTIES_LOADED_PAGINATED, GENES_PROPERTIES_ERROR,
   GENES_SELECT, GENES_DESELECT,
   GENES_EXPRESSION_LOADING, GENES_EXPRESSION_LOADED, GENES_EXPRESSION_ERROR,
-  MARKER_GENES_LOADING, MARKER_GENES_LOADED, MARKER_GENES_ERROR,
+  MARKER_GENES_LOADING, MARKER_GENES_LOADED, MARKER_GENES_ERROR, GENE_ORDER_SET,
```

Comment:
also, this sounds like it is better to be named to something `MARKER_GENE_ORDER_SET` and it definitely should be made clear in the code that it is for plots and tables. This doesn't look like the right place in the store for it